### PR TITLE
[Docs] TOC: Add section describing the use of children and grandchildren

### LINKED ISSUE #9165: [Docs] TOC: Add section describing the use of children and grandchildren
We need to document the use of the `grandchildren`<https://github.com/meshery/meshery/blob/03345f16a853f613b83b7d6e18c55552dd30b13e/docs/_data/toc.yml#L30-L31| stanza >that I added to the table of contents sidebar in the <https://docs.meshery.io/project/contributing/contributing-docs|Contributing to Docs> page.

[Slack Message](https://layer5io.slack.com/archives/C04LJS10VTL/p1697904313067449?thread_ts=1697904313.067449&cid=C04LJS10VTL)

### PR DESCRIPTION
**Pull request by coder12git**
**Notes for Reviewers**

This PR fixes #9165

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit e13fd4d19608d69721162416ba05601accc4387b at: https://6536ba44086c4f6e6e4ce620--meshery-docs-preview.netlify.app

**Comment by coder12git** [comment]
@leecalcote here it is :)

**Review by leecalcote — APPROVED** [review]

File: docs/pages/project/contributing/contributing-docs.md [hunk 1]
```diff
@@ -298,6 +298,48 @@ The output of the code snippet would be:
...
+      Thing 1
+       Subthing 1.1
+       Subthing 1.2
+      Thing 2
+      Thing 3
+```
+
+Here, **Subthing 1.1** and **Subthing 1.2** are the ``grandchildren`` of **Thing 1**.
+
+In the Sidebars, overall we have three sections -
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:335]:
```suggestion
In `docs/_includes/sidebar.html` contains three levels of navigation hierarchy.
```

File: docs/pages/project/contributing/contributing-docs.md [hunk 2]
```diff
@@ -298,6 +298,48 @@ The output of the code snippet would be:
...
+
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:336]: @coder12git, where does `links:` fit into all this?

Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:336]: @coder12git, where does `links:` fit into all this?

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 68fb82628c701bc709ede065b5c9db5317ade9e7 at: https://65375c6df75c7a28e22bfa0d--meshery-docs-preview.netlify.app

**Comment by saurabh100ni** [comment]
Let's discuss this on the Meshery dev call tomorrow at 7:30 pm IST / 9:00 am CT,
Please add this as your agenda item in Docs: https://docs.google.com/document/d/1Dg4Uz9rGTuGHmYZwIt47z1j5x3LmnpxD1M1SgypAkQ0/edit

File: docs/pages/project/contributing/contributing-docs.md [hunk 3]
```diff
@@ -298,6 +298,48 @@ The output of the code snippet would be:
...
+       Subthing 1.2
+      Thing 2
+      Thing 3
+```
+
+Here, **Subthing 1.1** and **Subthing 1.2** are the ``grandchildren`` of **Thing 1**.
+
+In `docs/_includes/sidebar.html` contains three levels of navigation hierarchy.
+
+- **Parent**: It serves as a top level category for related content.
```
Comment (leecalcote) [inline docs/pages/project/contributing/contributing-docs.md:337]: Parent isn't explicitly a thing, so please ensure that you are listing implicitly.

**Comment by leecalcote** [comment]
@coder12git should I continue to follow up on this PR or is it closing in favor of Debaditya's?

**Comment by coder12git** [comment]
> @coder12git should I continue to follow up on this PR or is it closing in favor of Debaditya's?

I think, both PR's are different, his PR is about alerts. html file, so i believe we shouldn't merge both in one PR, rest up to you, whatever you prefer it's fine from my end.

**Review by leecalcote — DISMISSED** [review]
Thank you for this, @coder12git 💯

**Comment by leecalcote** [comment]
@coder12git, will you follow up on the merge conflict at your convenience? Good to go afterwards.

**Comment by l5io** [comment]
🚀 Preview for commit e61a3de879ab0fc1add3b287ed4a2ccac0903e60 at: https://653ffa14c2f8ba2eb64769c3--meshery-docs-preview.netlify.app

**Pull request merged**